src/data/customer/Order.js

```diff
@@ -633,4 +640,7 @@ export const mergeLocationDataWithErrors = (errorContents, location) => {
       return errorList;
```

Comment:
Returning here will terminate the find arrow function not the mergeLocationDataWithErrors, I assume you want to return the errorList